libwallet/assets/ltc/sync.go

```diff
@@ -381,19 +380,20 @@ func (asset *Asset) CancelSync() {
 	asset.syncData.wg.Add(1)
 	go asset.stopSync()
 
+	// Indicate that the sync shutdown process is fully complete.
+	asset.EndSyncShuttingDown()
```

Comment:
it the same BTC sync.go

Reply:
Please find more info about it here: https://github.com/crypto-power/cryptopower/pull/563#issuecomment-2209125745

Reply:
see this comment
https://github.com/crypto-power/cryptopower/pull/563#discussion_r1666310189